### PR TITLE
add OCP as a target for deploying boosters

### PR DESCRIPTION
**Pull request by rhoads-zach**

### CODE REVIEW

File: docs/topics/proc-ocp-deploy-booster.adoc [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc new-project {project-name}
+----
+
+. Navigate to the root directory of your booster.
+
+. Deploy your booster to {OpenShiftContainerPlatform}.
+.. If your booster uses maven:
```
Comment (Ladicek) [inline docs/topics/proc-ocp-deploy-booster.adoc]: Should probably be upper-case Maven.

**Comment by rhoads-zach** [comment]
thanks for the feedback @Ladicek.  Fixed

File: docs/topics/booster-deploy-ocp.adoc [hunk 2]
```diff
@@ -0,0 +1,5 @@
+= Deploying your Booster to {OpenShiftContainerPlatform}
+
+IMPORTANT: In order to deploy a booster to {OpenShiftContainerPlatform}, you must first create it the {launcher} application in xref:oso-create-booster[{OpenShiftOnline}] or xref:osl-create-booster[a {OpenShiftLocal}].
```
Comment (Ladicek) [inline docs/topics/booster-deploy-ocp.adoc]: Sorry for my inability to find all issues in one go :-), but there's a word missing: "you must first create it the {launcher} application in {OpenShiftOnline}a {OpenShiftLocal}". It should (probably?) read "you must first create it _using_ the {launcher} application ...".

Comment (Ladicek) [reply docs/topics/booster-deploy-ocp.adoc]: Also I'm not sure in which context this text is used, but I guess it assumes that an existing Launcher (in OSO or Minishift) is used to create a .zip, but there's also an option of deploying Launcher to OCP just like you can deploy it to Minishift. Then, you can use it to directly launch a booster in the OCP instance.

File: docs/topics/booster-deploy-ocp.adoc [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
+To use the `oc` CLI Client with {OpenShiftContainerPlatform}, you follow the same steps as with your {OpenShiftLocal} but use the authentication token and URL from the {OpenShiftContainerPlatform} Web Console.
```
Comment (inoxx03) [inline docs/topics/booster-deploy-ocp.adoc]: > To use the `oc` CLI Client with {OpenShiftContainerPlatform}, you follow the same steps as with your {OpenShiftLocal} but use the authentication token and URL from the {OpenShiftContainerPlatform} Web Console.

IMHO, here you could omit the _you_ after the first comma.

File: docs/topics/circuit-breaker-mission-deploy-booster.adoc [hunk 4]
```diff
@@ -72,6 +77,4 @@ NAME                     HOST/PORT
 +
 A pod's route information gives you the base URL which you use to access it. In the example above, you would use `http://{app-name}-{project-name}.{osl-route-hostname}` as the base URL to access the applications.
```
Comment (inoxx03) [inline docs/topics/circuit-breaker-mission-deploy-booster.adoc]: I see that this is  not part of the diff, but we should avoid using the Saxon genitive in out docs. 
I'd change that to:

> The route information of a pod gives you the base URL that you use to access it. In the example above, you would use `http://{app-name}-{project-name}.{osl-route-hostname}` as the base URL to access the applications.

File: docs/topics/proc-ocp-deploy-booster.adoc [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+[sidebar]
+.Prerequisites
+--
+* Have a booster created using the {launcher} application in xref:oso-create-booster[{OpenShiftOnline}] or xref:osl-create-booster[a {OpenShiftLocal}].
+* Have access to an {OpenShiftContainerPlatform} Web console.
+* Have the `oc` CLI client installed.
+--
+
+.Procedure
+. Clone your booster's Github repository or download and extract the booster's ZIP.
```
Comment (inoxx03) [inline docs/topics/proc-ocp-deploy-booster.adoc]: nitpick: IMHO, there should be a comma before _or_.

File: docs/topics/proc-ocp-deploy-booster.adoc [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+$ mvn clean fabric8:deploy -Popenshift
+----
+.. If your booster uses `npm`.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ npm install && npm run openshift
+----
+
+. Open the `README.adoc` file in your booster's project and follow the instructions for interacting with your booster.
```
Comment (inoxx03) [inline docs/topics/proc-ocp-deploy-booster.adoc]: Not really a problem, but maybe we could make this more specific:

> Open the `README.adoc` file located in the project containing your booster source, and follow the instructions for interacting with your booster.

**Comment by rhoads-zach** [comment]
thanks @inoxx03 and @Ladicek for the feedback

Updated.

**Pull request merged**